Kudu.Core/Infrastructure/OperationClient.cs

```diff
@@ -48,5 +50,13 @@ public OperationClient(ITracer tracer)
                 }
             }
         }
+
+        public static void SkipSslValidation()
```

Comment:
No preference on name.   I think what you proposed `SkipSslValidationIfNeeded` is fine.   Will fix it.